app/navigation/index.js

```diff
@@ -20,9 +15,7 @@ const store = createStore()
 class App extends Component {
   render () {
     return (
-      <Provider store={store}>
-        <RootContainer />
```

Comment:
`RootContainer` ma w sobie też nawigację, ale nie widzę, zebyś ją przepiął gdziekolwiek. To celowo?